library/src/main/java/com/google/maps/android/clustering/view/ClusterRenderer.java

```diff
@@ -51,6 +51,11 @@
      */
     void setAnimation(boolean animate);
 
+    /**
+     * Sets the length of the animation. The default duration is 300 milliseconds.
+     */
+    void setAnimationDuration(int animationDuration);
```

Comment:
`ValueAnimator.setDuration()` takes in a `long`, so please use a `long` value here and throughout too.